test/topology_experimental_raft/test_mv_tablets.py

```diff
@@ -7,14 +7,74 @@
+# read the replicas information from the same server (it takes time for this
+# information to propagate to all servers).
+async def get_tablet_replicas(manager: ManagerClient, server: ServerInfo, keyspace_name: str, table_or_view_name: str, token: int):
+    host = (await wait_for_cql_and_get_hosts(manager.cql, [server], time.time() + 60))[0]
+    await read_barrier(manager.cql, host)
+
+    rows = await manager.cql.run_async(f"SELECT last_token, replicas FROM system.tablets where "
+                                       f"keyspace_name = '{keyspace_name}' and "
+                                       f"table_name = '{table_or_view_name}'"
+                                       " ALLOW FILTERING", host=host)
+    for row in rows:
+        if row.last_token >= token:
+            return row.replicas
+
+# This convenience function assumes a table has RF=1 and only a single tablet,
+# and moves it to one specific node "server" - and pins it there (disabling
+# further tablet load-balancing). It is not specified which *shard* on that
+# node will receive the tablet.
+async def pin_the_only_tablet(manager, keyspace_name, table_or_view_name, server):
+    # We need to send load-balancing commands to one of the nodes and they
+    # will be propagated to all of them. Since we already know of
+    # target_server, let's just use that.
+    await manager.api.disable_tablet_balancing(server.ip_addr)
+    tablet_token = 0 # Doesn't matter since there is one tablet
+    source_replicas = await get_tablet_replicas(manager, server, keyspace_name, table_or_view_name, tablet_token)
+    # We assume RF=1 so get_tablet_replicas() returns just one replica
+    assert len(source_replicas) == 1
+    source_host_id, source_shard = source_replicas[0]
+
+    target_host_id = await manager.get_host_id(server.server_id)
+    target_shard = 0 # We don't care which shard to use
+
+    # Currently migrating a tablet in the same node is not allowed.
+    # We need to just do nothing in this case - the tablet is already in
+    # its desired node (and we didn't specify which shard is desired).
+    # The str() is needed because we can't compare HostId to string :-(
+    if str(target_host_id) == str(source_host_id):
+        return
+
+    # Finally move the tablet. We can send the command to any of the hosts,
+    # it will propagate it to all of them.
+    await manager.api.move_tablet(server.ip_addr, keyspace_name, table_or_view_name, source_host_id, source_shard, target_host_id, target_shard, tablet_token)
+
+# Assert that the given table uses tablets, and has only one. It helps
+# verify that a test that attempted to enable tablets - and set up only
+# one tablet for the entire table - actually succeeded in doing that.
+async def assert_one_tablet(cql, keyspace_name, table_or_view_name):
+    rows = await cql.run_async(f"SELECT last_token, replicas FROM system.tablets where keyspace_name = '{keyspace_name}' and table_name = '{table_or_view_name}' ALLOW FILTERING")
```

Comment:
Again, maybe it will be useful, maybe it won't.

Most importantly, as you said, if you wanted to move something into a library, it shouldn't be the specific assert_one_tablet() but rather something more general to enquire about tablets - basically just the one run_async line. If I were to move assert_one_tablet() into a library the result would be - again - dtest - a big mess where if the test fails you don't understand what it does because all it does is to call into utility functions you are not familiar with and are probably buggy themselves.